jenkins/job-configs/kubernetes-jenkins-pull/fejta-pull.yaml

```diff
@@ -41,9 +41,9 @@
             fail: true
     builders:
         - shell: |
-            # TODO(fejta): change this back to kubernetes/test-infra -b master
-            git clone https://github.com/fejta/test-infra -b fejta
-            './test-infra/jenkins/bootstrap.py' --job='{job-name}' --repo=kubernetes/kubernetes --pull="${{ghprbPullId}}"
+            # TODO(fejta): consider a stable tag instead of master
+            git clone https://github.com/kubernetes/test-infra -b master
+            './test-infra/jenkins/bootstrap.py' --job='{job-name}' --repo=kubernetes/kubernetes --pull="${{ghprbPullId}}" --root="${{GOPATH}}/src/github.com"
```

Comment:
did you intend to make it so unit and e2e runs also checkout to this path? (I approve, just wanted to check that this was intentional.)